README.md

```diff
@@ -8,6 +8,7 @@ The Ansible Datadog role installs and configures the Datadog Agent and integrati
 
 - Requires Ansible v2.6+.
 - Supports most Debian and RHEL-based Linux distributions, and Windows.
+- When using Ansible 2.10+, requires the `ansible.windows` collection to be installed (even on Linux): `ansible-galaxy collection install ansible.windows`
```

Comment:
nit: I would move the command to a code block, I think it is more readable that way